test/into.js

```diff
@@ -29,6 +29,12 @@ describe('into', function() {
         assert.deepEqual(R.into({}, R.identity, [{a: 1}, {b: 2, c: 3}]), {a: 1, b: 2, c: 3});
     });
 
+    it('dispatches to objects that implement `reduce`', function() {
+        var obj = {x: [1, 2, 3], reduce: function(f, acc) { return R.reduce(f, acc, this.x); }};
+        assert.deepEqual(R.into([], R.map(add(1)), obj), [2, 3, 4]);
+        assert.deepEqual(R.into([], R.map(add(1)), obj), [2, 3, 4]);
+    });
+
```

Comment:
would probably be good to test one that does not call `R.reduce` but some other `reduce`